### ISSUE TITLE
Cors issue 

### ISSUE THREAD

**Issue by gk0307** [apache/pulsar#19694]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### Version

using the 2.10 version

### Minimal reproduce step

We are creating a react Application for publishing the message in the topic using the post **https://pulsar.apache.org/docs/next/client-libraries-rest/**   method but we are facing a Cors issue. Tried to fix that issue by changing the configuration files using the following steps:

1. we are changing the **broker.config** file add set Access-Control-Allow-Origin "*".
2. Also change **standalone.conf**  file add `allowedOrigins=http://192.168.0.181:8080/, http://192.168.0.181:5174/` and:

```
# Enable the WebSocket API service in broker
webSocketServiceEnabled=true
```

But we can't publish the message on that topic.
 
Error:
````
Access to XMLHttpRequest at 'http://192.168.0.181/topics/persistent/dev/compsource/policy_change_stat' from origin 'http://192.168.0.181:5174' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
````
Request Header:
Request URL: http://192.168.0.181/topics/persistent/dev/compsource/policy_change_stat
Referrer Policy: strict-origin-when-cross-origin

### What did you expect to see?

It should allow  to connect and publish the message in that topic

### What did you see instead?

Getting cors error in the post method when we send to the pulsar topic

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by tisonkun** [comment]
@gk0307 Can you provide a minimal reproduce repo? I don't find `Access-Control-Allow-Origin` config in broker.conf and also other configs.

**Comment by tisonkun** [comment]
Closing as stale. If it's still relevant to maintained versions, feel free to open a new issue with reproduce.
